README.md

```diff
@@ -510,6 +510,7 @@ Table of Contents
   * [knapsackpro.com](https://knapsackpro.com) - Speed up your tests with optimal test suite parallelisation on any CI provider. Split Ruby, JavaScript tests on parallel CI nodes to save time. Free plan for up to 10 minutes test files and free unlimited plan for Open Source projects.
   * [webhook.site](https://webhook.site) - Verify webhooks, outbound HTTP requests, or emails with a custom URL.  Temporary URL and email address is always free.
   * [Vaadin](https://vaadin.com) — Build scalable UIs in Java or TypeScript, and use the integrated tooling, components and design system to iterate faster, design better and simplify the development process. Unlimited Projects with 5 years free maintenance.
+  *   * [Foresight](https://thundra.io/foresight) — Thundra Foresight is a tool for debugging and troubleshooting test failures in no time and optimizing build duration and performance in your CI pipeline.
```

Comment:
All I can find regarding pricing is "free for a limited time" what does that mean? Do you have a link to detailed pricing indicating the free tier details?

Assujming you do - please also remove the extra `*`

Reply:
I removed the extra `*` 
Currently, we don't have pricing plans and other limitations. It'll stay as free for a while, everyone has full access to all features. We won't charge our users suddenly; we are not taking any credit card information, etc. Apart from that, it will be forever free for open-source projects.

Reply:
That’s great. But I need to see this written down somewhere on the page in question.  If you add about free forever for OSS somewhere on tbe site we can mergr it.

Reply:
Thanks! You can see our open-source policy in our docs: https://foresight.docs.thundra.io/foresight-for-open-source